### ISSUE TITLE
Cannot find tests with junit 5

### ISSUE THREAD

**Issue by kbradl16** [mjedynak/pit-idea-plugin#46]
I'm using maven with using org.junit:junit-bom:5.6.2 
Not sure if this is the issue but the testPlugin is set to junit. Not sure what it should be.

**My PIT Runner configuration settings:**
Target classes: com.company.project.packageToTest
Target tests: com.company.project.packageToTest
Source dir: C:/folder/project/src/main/java
Report dir: C:/folder/project/target/report
Other params: --outputFormats XML,HTML --verbose=true

```
3:19:04 PM PIT >> INFO : Sending 198 test classes to minion
3:19:04 PM PIT >> INFO : Sent tests to minion
3:19:04 PM PIT >> INFO : MINION : 3:19:04 PM PIT >> FINE : Expecting 198 tests classes from parent
3:19:04 PM PIT >> FINE : Tests classes received

3:19:04 PM PIT >> INFO : MINION : 3:19:04 PM PIT >> INFO : Checking environment

3:19:05 PM PIT >> INFO : MINION : 3:19:05 PM PIT >> INFO : Found  0 tests

3:19:07 PM PIT >> INFO : MINION : 3:19:07 PM PIT >> INFO : Dependency analysis reduced number of potential tests by 0

3:19:07 PM PIT >> INFO : MINION : 3:19:07 PM PIT >> INFO : 0 tests received

3:19:07 PM PIT >> INFO : MINION : 3:19:07 PM PIT >> FINE : Running 0 units
```

**Comment by kbradl16** [comment]
Awkward. I think i figured out the issue. I needed to import pitest-junit5-plugin as a direct import. And add `--testPlugin=junit5` to "Other params"
```
<dependency>
            <groupId>org.pitest</groupId>
            <artifactId>pitest-junit5-plugin</artifactId>
            <version>${mutation.pitest.junit5.version}</version>
        </dependency>
```

**Comment by mjedynak** [comment]
fixed in 1.4.5
